### PR TITLE
Deed helper

### PR DESCRIPTION
**Pull request by Phemus**

### CODE REVIEW

File: app/case/deed_created.mustache [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<main id="content" role="main">
+
+  <div class="inner-block">
```
Comment (endamccormack) [inline app/case/deed_created.mustache]: this div and its close div are not on the same indent.

File: app/case/views.py [hunk 2]
```diff
@@ -11,6 +11,9 @@ def __init__(self, cases):
 class Start(Template):
     pageTitle = "View your case list"
```
Comment (endamccormack) [inline app/case/views.py:13]: have you ran flake8? I think it would complain about 1 line after class.

File: app/service/case_api.py [hunk 3]
```diff
@@ -16,7 +16,7 @@ def get_cases(self):
     def create_case(self, deed_id):
         payload = {
             "conveyancer_id": 1,
-            "deed_id": int(deed_id)
+            "deed_id": 1
```
Comment (endamccormack) [inline app/service/case_api.py]: you removed the need for deed_id but its still in the method signature

File: app/service/case_api.py [hunk 4]
```diff
@@ -24,3 +24,15 @@ def create_case(self, deed_id):
             return case_from_json(response.json())
         else:
             response.raise_for_status()
+
+    def update_case_with_deed(self, deed_id, case_id):
+        payload = {
+            "deed_id": deed_id
+        }
+        endpoint = self.case_endpoint + "/" + case_id + "/deed"
```
Comment (michaeldfallen) [inline app/service/case_api.py]: Can you use string formatting over concatenation? It prevents some obscure type problems that can occur:

```
"{case}/{case_id}/deed".format(
    case=self.case_endpoint,
    case_id=case_id
)
```

File: app/case/routes.py [hunk 5]
```diff
@@ -29,3 +29,10 @@ def create_case():
     def start_page():
 
         return views.Start().render()
+
+    @blueprint.route('/case/<case_id>/deed/new')
```
Comment (michaeldfallen) [inline app/case/routes.py]: The create deed transaction is going to be a full protracted transaction for conveyancers so can you move this in to a new blueprint that will contain all the "create deed" stuff.

**Comment by michaeldfallen** [comment]
When we were talking about the journey last we talked about how it's going to become a new full transaction with multiple steps. So it would be useful to follow the `click create deed` -> `Taken to create deed transaction` -> `click create deed` -> `return to caselist` flow. That way it's easy to slip steps in to the middle of that flow.

File: app/case/views.py (hunk 2 above)
Comment (Phemus) [inline app/case/views.py:13]: I have flake8 installed in my ide and it didn't pick this one up but will check it

File: app/deed/deed_created.mustache [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<main id="content" role="main">
+
+  <div class="inner-block">
```
Comment (endamccormack) [inline app/deed/deed_created.mustache]: This is under indented

**Comment by michaeldfallen** [comment]
:+1: Merging

**Pull request merged**